network-api/networkapi/buyersguide/templates/buyersguide/bg_base.html

```diff
@@ -0,0 +1,112 @@
+{% extends "pages/base.html" %}
+
```

Comment:
This is the new buyers guide base template. Its mostly a duplicate of the original. 

Reminder: We're doing this to namespace templates a bit better, and to not change anything thats in product so we can test things thoroughly before merging anything into master.